### ISSUE TITLE
[BUG] Error during test execution: KeyError: 'schema' in get_response_schema

### ISSUE THREAD

**Issue by DavidFarago** [schemathesis/schemathesis#1220]
**Describe the bug**
When calling `schemathesis run -H "Authorization: Bearer <bearerToken>" --checks all --base-url=<baseURL> --show-errors-tracebacks`, I get the following error:
```
Traceback (most recent call last):
  File "/home/davef/anaconda3/lib/python3.7/site-packages/schemathesis/runner/impl/core.py", line 446, in network_test
    max_response_time,
  File "/home/davef/anaconda3/lib/python3.7/site-packages/schemathesis/runner/impl/core.py", line 495, in _network_test
    run_checks(case, checks, check_results, result, response, context.response_time * 1000, max_response_time)
  File "/home/davef/anaconda3/lib/python3.7/site-packages/schemathesis/runner/impl/core.py", line 324, in run_checks
    skip_check = check(response, case)
  File "/home/davef/anaconda3/lib/python3.7/site-packages/schemathesis/specs/openapi/checks.py", line 103, in response_schema_conformance
    return case.operation.validate_response(response)
  File "/home/davef/anaconda3/lib/python3.7/site-packages/schemathesis/models.py", line 682, in validate_response
    return self.schema.validate_response(self, response)
  File "/home/davef/anaconda3/lib/python3.7/site-packages/schemathesis/specs/openapi/schemas.py", line 378, in validate_response
    scopes, schema = self.get_response_schema(definition, operation.definition.scope)
  File "/home/davef/anaconda3/lib/python3.7/site-packages/schemathesis/specs/openapi/schemas.py", line 635, in get_response_schema
    return scopes, to_json_schema_recursive(option["schema"], self.nullable_name)
KeyError: 'schema'
```

My OpenAPI spec has many return code objects that contain 
```
            "content": {
              "application/json": {
                "examples": {
                  ...
                }
             }
           }
```
i.e. without `schema` object. The OpenAPI spec does pass its schema validation.

**Expected behavior**
No ERRORs during test execution (or invalid schema notification for the OpenAPI spec).

**Environment (please complete the following information):**
-   OS: Ubuntu
-   Python version: 3.7.6
-   Schemathesis version: 3.8.0
-   Spec version: 3.0.1 OpenAPI json

**Comment by Stranger6667** [comment]
Hi @DavidFarago 

Thank you very much for reporting :) Indeed, I missed that and will work on the fix soon.

**Comment by Stranger6667** [comment]
The fix is released in `3.9.4`. Let me know if it doesn't solve the issue :)
